.changeset/great-bobcats-suffer.md

```diff
@@ -0,0 +1,26 @@
+---
+"@razorpay/blade": minor
```

Comment:
Should we make a major version bump? Even though its a breaking change, I'd prefer not bumping the major version since we're at an early stage anyway